Effects/Parallax/source/Flake.hx

```diff
@@ -0,0 +1,74 @@
+package;
+
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.math.FlxMath;
+import flixel.math.FlxRandom;
+import flixel.util.FlxColor;
+
+class Flake extends FlxSprite
+{
+	private var _level:Int = 0;
+	private var _initialized:Bool = false;
+	
+	
+	public function new(Level:Int = 0) 
+	{
+		
+		super();
+		_level = Level;
+		revive();
+		_initialized = true;
+	}
+	
+	override public function revive():Void 
+	{
+		var size:Int = Std.int(FlxMath.bound(_level / 2, 1, 2));
+		makeGraphic(size, size, FlxColor.WHITE.getDarkened(.5 - (_level * .06)));
```

Comment:
The point is for recycling objects, I want to do as much as I can in `revive` so that it's more likely to generate a totally different looking flake, and put it in a different spot.
True, most people would never notice if the flakes were always the same, but I wanted to be thorough.